frontend/src/elm/Login/Major/Models.elm

```diff
@@ -0,0 +1,22 @@
+module Login.Major.Models exposing (..)
+
+
+type alias Major =
+  { majorType : String
+  , majorName : String
+  }
+
+
+type alias MajorForm =
```

Comment:
https://github.com/Gizra/elm-hedley
This project names as `Model` in every components.
And the reason why they use `Model` instead of specific name is shown [here](https://github.com/Gizra/elm-hedley/blob/master/src/elm/App/Model.elm):
the code that references model is self-explanatory.